apps/wallet/src/ui/app/components/menu/content/AccountActions.tsx

```diff
@@ -58,6 +59,12 @@ export function AccountActions({ account }: AccountActionsProps) {
     return (
         <div className="flex flex-row items-center flex-1 gap-1">
             {actionContent}
+            <Link
```

Comment:
Already discussed in slack just noting it here as well. We should only show the `Export Passphrase` button only for derived accounts